Workbooks/SapMonitor/SapHanaInfrastructure/SapHanaInfrastructure.workbook

```diff
@@ -0,0 +1,376 @@
+                    "thresholdValue": "70",
+                    "representation": "success",
+                    "text": "{0}{1}"
+                  },
+                  {
+                    "operator": ">",
+                    "thresholdValue": "70",
+                    "representation": "2",
+                    "text": "{0}{1}"
+                  },
+                  {
+                    "operator": ">=",
+                    "thresholdValue": "85",
+                    "representation": "critical",
+                    "text": "{0}{1}"
+                  },
+                  {
+                    "operator": "Default",
+                    "thresholdValue": null,
+                    "representation": "success",
+                    "text": "{0}{1}"
+                  }
+                ]
+              }
+            },
+            {
+              "columnMatch": "Disk_Used",
+              "formatter": 18,
+              "formatOptions": {
+                "min": 0,
+                "max": 100,
+                "palette": "red",
+                "showIcon": true,
+                "thresholdsOptions": "icons",
+                "thresholdsGrid": [
+                  {
+                    "operator": "<=",
+                    "thresholdValue": "70",
+                    "representation": "success",
+                    "text": "{0}{1}"
+                  },
+                  {
+                    "operator": ">",
+                    "thresholdValue": "70",
+                    "representation": "2",
+                    "text": "{0}{1}"
+                  },
+                  {
+                    "operator": ">",
+                    "thresholdValue": "80",
+                    "representation": "4",
+                    "text": "{0}{1}"
+                  },
+                  {
+                    "operator": "Default",
+                    "thresholdValue": null,
+                    "representation": "success",
+                    "text": "{0}{1}"
+                  }
+                ]
+              }
+            },
+            {
+              "columnMatch": "NW_IN",
+              "formatter": 4,
+              "formatOptions": {
+                "palette": "orange",
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "NW_OUT",
+              "formatter": 4,
+              "formatOptions": {
+                "showIcon": true
+              }
+            }
+          ],
+          "labelSettings": []
+        }
+      },
+      "name": "query - 6"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": " CPU Utilization for HANA host"
+      },
+      "name": "text - 1"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "SapHana_LoadHistory_CL\r\n| summarize avg(CPU_d) by HOST_s , bin(UTC_TIMESTAMP_t, 15m)",
+        "size": 0,
+        "timeContext": {
```

Comment:
it looks like some of the steps have an explicit time range set, and others are using "set in query", but then don't actually have a time range filter in the query.  without an explicit time range set in the query, it will default to last 24h.

you might want to add a time range parameter at the top and have all the queries use that time range parameter?

Reply:
Sure; the current template is really just a "placeholder" to test the integration.
Will follow your suggested design patterns and add a time range picker at the top.